gui/l10n/AbracaDABra_de.ts

```diff
@@ -3756,57 +3756,57 @@ Decoded %1 / %2 MOT objects</source>
     </message>
     <message>
         <location filename="../tii/scannerdialog.cpp" line="219"/>
         <source>Hide local (known) transmitters</source>
-        <translation type="unfinished"></translation>
+        <translation>Blende lokale (bekannte) Sendeanlagen aus</translation>
     </message>
     <message>
         <location filename="../tii/scannerdialog.cpp" line="231"/>
         <source>AutoSave CSV</source>
-        <translation type="unfinished"></translation>
+        <translation>Autom. Speichern CSV</translation>
     </message>
     <message>
         <location filename="../tii/scannerdialog.cpp" line="239"/>
         <source>Save as CSV</source>
-        <translation type="unfinished"></translation>
+        <translation>Speichern als CSV</translation>
     </message>
     <message>
         <location filename="../tii/scannerdialog.cpp" line="243"/>
         <source>Load from CSV</source>
-        <translation type="unfinished"></translation>
+        <translation>Lade von CSV</translation>
     </message>
     <message>
         <location filename="../tii/scannerdialog.cpp" line="248"/>
         <source>Clear scan results</source>
-        <translation type="unfinished"></translation>
+        <translation>Scanergebnisse leeren</translation>
     </message>
     <message>
         <location filename="../tii/scannerdialog.cpp" line="254"/>
         <source>Do you want to clear scan results?</source>
-        <translation type="unfinished"></translation>
+        <translation>Wollen Sie wirklich die Scanergebnisse leeren?</translation>
     </message>
     <message>
         <location filename="../tii/scannerdialog.cpp" line="256"/>
         <source>You will loose current scan results, this action is irreversible.</source>
-        <translation type="unfinished"></translation>
+        <translation>Sie werden die Scanergebnisse verlieren, diese Aktion ist unumkehrbar.</translation>
     </message>
     <message>
         <location filename="../tii/scannerdialog.cpp" line="272"/>
         <source>Clear local (known) transmitter database</source>
-        <translation type="unfinished"></translation>
+        <translation>Datenbank lokaler (bekannter) Sendeanlagen leeren</translation>
     </message>
     <message>
         <location filename="../tii/scannerdialog.cpp" line="278"/>
         <source>Do you want to clear local (known) transmitter database?</source>
-        <translation type="unfinished"></translation>
+        <translation>Wollen Sie wirklich die Datenbank lokaler (bekannter) Sendeanlagen leeren?</translation>
     </message>
     <message>
         <location filename="../tii/scannerdialog.cpp" line="280"/>
         <source>You will loose all records in the database, this action is irreversible.</source>
-        <translation type="unfinished"></translation>
+        <translation>Sie werden alle Einträgen in der Datenbank verlieren, diese Aktion ist unumkehrbar.</translation>
```

Comment:
typo: Einträgen --> Einträge

Reply:
Sorry, I was in a hurry, just in the last moment before I went to holiday...